cmd/state/exec3/state.go

```diff
@@ -270,6 +257,9 @@ func (rw *Worker) RunTxTaskNoLock(txTask *state.TxTask) {
 		}
 
 		// MA applytx
+		if rw.chainConfig.Optimism != nil {
+			rw.evm.Context.L1CostFunc = opstack.NewL1CostFunc(rw.chainConfig, rw.ibs)
+		}
```

Comment:
It seems like we're setting L1CostFunc at every places where we use optimism evm (i.e. otterscan apis, debug apis, etc). 
Instead, what do you think about doing this in the initializer of EVMContext by checking the `chainConfig.optimism` so that we don't need this repetitive code everywehere?

Reply:
Honestly not sure why we did take this approach, but sounds good to me!